utilities.js

```diff
@@ -33,13 +33,30 @@ exports.convertModelJsonToUIJson = function(filepath) {
     ui = {}
     ui.name = ""
     ui.data = []
+    ui.options = []
+    numOfRegsRead = 0
     model.devices.forEach(device => {
-        device.registers.forEach(reg => {
+        device.registers.forEach((reg, index) => {
+            let properties = {}
             if(reg.dataType.wordLength == 1){
                 properties = {
                     enumeration: ["Disable", "Enable"]
                 }
             }
+            else if(reg.enumerations){
+                option = {
+                    data: [index + numOfRegsRead],
+                    enumerations: Object.keys(reg.enumerations).map(key => {
+                        let result = {
+                            "key": key,
+                            "value": reg.enumerations[key]
+                        }
+                        return result
+                      })
+                }
+                properties = {enumerations: []}
```

Comment:
What's the purpose of this? The next line seems to push the enumerations created before this to the ui object.